src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -2796,7 +2796,7 @@ private DatasetVersion getDatasetVersionOrDie(final DataverseRequest req, String
 
             @Override
             public Command<DatasetVersion> handleLatest() {
-                return new GetLatestAccessibleDatasetVersionCommand(req, ds, includeDeaccessioned);
+                return new GetLatestAccessibleDatasetVersionCommand(req, ds, includeDeaccessioned, checkPermsWhenDeaccessioned);
```

Comment:
If the latest published version is also deaccessioned and there's a previous published version that is not deaccessioned should I get the previous version when I call this command? In other words version 2 is deaccessioned but version 1 is not, if I call this command will I get version 1 or null? (I don't see a test case for this in DatasetsIT

Reply:
- If you are an authenticated user and have access to the draft and a draft exists you will get the draft. 
- If there is no draft or you don't have access and you set the parameter to getDeaccessioned to true you should get version 2
- If there is no draft or you don't have access and you set the parameter to getDeaccessioned to false you should get version 1

I can add this case if you think it would be helpful I just was not sure if this test had already too much going on, at some point I was thinking about splitting it into different testing for latest, latest-published and specific versions.